tests/api/conftest.py

```diff
@@ -27,22 +27,25 @@ def client():
 
 
 @pytest.fixture
-def table():
-    table_properties = get_table_properties_from_template()
+def tables():
     with mock_dynamodb2():
```

Comment:
I slightly prefer separate `job_table` and `user_table` mocks, we might only have 1 test that uses both